Lib/sqlite3/__main__.py

```diff
@@ -35,57 +39,65 @@ def runpy(self, source, filename="<input>", symbol="single"):
 
     def printhelp(self, ignored):
         print("Enter SQL code and press enter.")
+        return None
```

Comment:
It can be removed too. It is an anti-pattern.

And maybe the function could be inlined.

Reply:
I thought about inlining it as well. Let's do that for the initial version. If we need to expand it, we just change the implementation.